metricflow/model/validations/validator_helpers.py

```diff
@@ -197,7 +199,7 @@ class ValidationFutureError(ValidationIssue, BaseModel):
     def level(self) -> ValidationIssueLevel:  # noqa: D
         return ValidationIssueLevel.FUTURE_ERROR
 
-    def as_readable_str(self, verbose: bool = False) -> str:
+    def as_readable_str(self, verbose: bool = False, prefix: Optional[str] = None) -> str:
         """Return a easily readable string that can be used to log the issue."""
         return (
             f"{super().as_readable_str(verbose=verbose)}"
```

Comment:
We now have to pass the prefix in the call to `super().as_readable_str` so that when `as_cli_formatted_str` gets called, future errors get the coloration.
```suggestion
            f"{super().as_readable_str(verbose=verbose, prefix=prefix)}"
```

Reply:
Duh. Nice catch, thanks!